src/main/java/emu/grasscutter/command/commands/GraduateCommand.java

```diff
@@ -0,0 +1,160 @@
+package emu.grasscutter.command.commands;
+
+import emu.grasscutter.Grasscutter;
+import emu.grasscutter.command.Command;
+import emu.grasscutter.command.CommandHandler;
+import emu.grasscutter.game.player.Player;
+import emu.grasscutter.utils.Language;
+import java.util.*;
+
+@Command(label = "graduate", usage = "graduate userid", aliases = {"grad"}, permission = "player.graduate", permissionTargeted = "player.graduate.others")
+public class GraduateCommand extends GiveArtifactCommand {
+    @Override
+    public void execute(Player sender, Player targetPlayer, List<String> args) {
+        try {
+            Player toWho = null;
+            if(targetPlayer != null)
+            {
+                toWho = targetPlayer;
+            }
+            else if(sender != null)
+            {
+                toWho = sender;
+            }
+            else
+            {
+                CommandHandler.sendMessage(sender, Language.translate("commands.execution.need_target"));
+                return;
+            }
+            String[][] graduateData = new String[][]{
+// 神里绫人毕业圣遗物
```

Comment:
Please use English in comments if possible

Reply:
Translation was updated